src/scalarstats.jl

```diff
@@ -166,16 +206,40 @@ nquantile{T<:Real}(v::AbstractArray{T}, n::Integer) = quantile(v, (0:n)/n)
 #############################
 
 # span, i.e. the range minimum(x):maximum(x)
+"""
+    span(x) -> Range
+
+Return the span of an integer array, i.e. the range `minimum(x):maximum(x)`.
+"""
 span{T<:Integer}(x::AbstractArray{T}) = ((a, b) = extrema(x); a:b)
 
 # Variation coefficient: std / mean
+"""
+    variation(x[, m]) -> Float
```

Comment:
@nalimilan Come to think of it, do we want the return type listed for everything? Looks like the majority of Julia documentation across various packages doesn't list it anyway; it uses `->` occasionally but more often than not omits it entirely. What would you think of that?